azul/src/display_list.rs

```diff
@@ -1752,7 +1754,18 @@ fn push_background(
                     None => TypedSize2D::new(image_dimensions.0, image_dimensions.1)
                 };
 
-                push_image(info, builder, app_resources, image_id, size);
+                let info = if let Some(repeat) = background_repeat {
+                    match repeat {
+                        StyleBackgroundRepeat::NoRepeat => LayoutPrimitiveInfo::with_clip_rect(info.rect, TypedRect::from_size(size)),
```

Comment:
I would just insert a `use StyleBackgroundRepeat::*` here to make the line shorter (so that you can simplify the `StyleBackgroundRepeat::NoRepeat` to just `NoRepeat`). Since imports are scoped, it shouldn't affect the surrounding code. It's just a minor nitpick though.